src/Chapter06/Listing06.20.ProvidingPropertyValidation.cs

```diff
@@ -24,7 +25,7 @@ public string LastName
                 if(value == null)
                 {
                     // Report error
-                    throw new ArgumentNullException(nameof(value));
+                    throw new ArgumentNullException("value");
```

Comment:
Why change away from nameof?

Reply:
Updated to match book's text. I'll update everything to be nameof() in a future version